### ISSUE TITLE
【功能请求】单独代理设置。

### ISSUE THREAD

**Issue by tensor-tech** [dice2o/BingGPT#22]
能不能给软件添加一个单独的代理设置，例如proxy=http://127.0.0.1:7890

**Comment by dice2o** [comment]
I don't think this option is necessary, the app will use system's proxy settings. You just need to add bing.com and its subdomains to the rules in your VPN.

**Comment by QwindF** [comment]
If possible, I also hope to add this function, thank you

**Comment by hnuzhoulin** [comment]
If possible, I also recommend adding this feature, system proxy has a large impact area, generally I do not set global proxy.
Thanks.

**Comment by Sirlanri** [comment]
I think this function is pretty good if possible:)

**Comment by Cooky-12** [comment]
好像 chrome 的命令参数可以用
`BingGPT.exe --proxy-server=127.0.0.1:1080`

you can try chrome arguments
`BingGPT.exe --proxy-server=127.0.0.1:1080`

**Comment by DoveBoy** [comment]
希望也能直接加一个自定义proxy的功能，目前自己使用scoop安装的，启动参数不知道怎么加
